### PR TITLE
Update troubleshooting guide

### PR DESCRIPTION
**Pull request by jobannon**
# Summary
- Add Troubleshooting Section Regarding a clean upgrade/uninstall (scorched earth)
- Adapted from the below issue
 https://github.com/FusionAuth/fusionauth-issues/issues/1143#issue-830989649

### CODE REVIEW

File: site/docs/v1/tech/troubleshooting.adoc [hunk 1]
```diff
@@ -142,6 +143,65 @@ If you can still get a lower number of logins per second than you'd expect with
 
 If the MD5 configuration allows you to achieve a much higher logins per second, then the CPU is your bottleneck. If you are CPU bound, the only way to get more logins per second is to horizontally scale (add more nodes) or vertically scale (use larger CPUs with each node).
 
+
+== Troubleshooting Uninstall-Upgrade-Reinstall on RPMs
```
Comment (mooreds) [inline site/docs/v1/tech/troubleshooting.adoc:147]: I'm not sure 'on RPMs' makes sense. Maybe 'with RPMs'?

File: site/docs/v1/tech/troubleshooting.adoc [hunk 2]
```diff
@@ -142,6 +143,65 @@ If you can still get a lower number of logins per second than you'd expect with
...
+** (is also safe to delete, but is not necessary)
+
+1. Run a re-install of FusionAuth App and FusionAuth Search
+* Using the RPM commands
+** `sudo rpm -i <file_to_be_installed>.rpm`
+** link:/direct-download[Direct download packages/rpm link]
+* If on a development machine, FastPath installation is also an option.
+**  link:/docs/v1/tech/installation-guide/fast-path/#fast-path-install[Fast Path Install]
+
+1. This should leave you with a clean install preserving:
```
Comment (mooreds) [inline site/docs/v1/tech/troubleshooting.adoc:189]: This seems like burying the lede. Move this up to the top, as this is the entire goal, correct?

File: site/docs/v1/tech/troubleshooting.adoc [hunk 3]
```diff
@@ -142,6 +143,65 @@ If you can still get a lower number of logins per second than you'd expect with
...
+
+== Troubleshooting Uninstall-Upgrade-Reinstall on RPMs
+
+[WARNING]
+====
+This will completely remove FusionAuth App and FusionAuth Search from your installation. +
+Proceed with caution. It is recommended you back up the database before performing any troubleshooting steps listed here.
+====
+
+=== The Nuclear Option
```
Comment (mooreds) [inline site/docs/v1/tech/troubleshooting.adoc:155]: Sorry to be a downer but this is a bit too colorful for docs. Can you make it more ... boring :) please?

**Review by mooreds — CHANGES_REQUESTED** [review]
A few changes, but going to helpful to folks.

File: site/docs/v1/tech/troubleshooting.adoc [hunk 4]
```diff
@@ -142,6 +143,65 @@ If you can still get a lower number of logins per second than you'd expect with
...
+- `sudo rm -rf /usr/local/fusionauth/fusionauth-search/`
+- `sudo rm -rf /usr/local/fusionauth/fusionauth-app/`
+
+1. After removal, the following files will remain:
+* `/usr/local/fusionauth/config`
+* `/usr/local/fusionauth/data`
+* `/usr/local/fusionauth/java`
+* `/usr/local/fusionauth/bin`
+** (if it exists)
+* `/usr/local/fusionauth/java`
```
[the remark this answers is not in the document]
Comment (jobannon) [reply site/docs/v1/tech/troubleshooting.adoc:179]: I think it was an oversight - I will remove it.

File: site/docs/v1/tech/troubleshooting.adoc (hunk 3 above)
Comment (jobannon) [reply site/docs/v1/tech/troubleshooting.adoc:155]: 💤 🛌🏻  🙂

File: site/docs/v1/tech/troubleshooting.adoc [hunk 5]
```diff
@@ -142,6 +143,65 @@ If you can still get a lower number of logins per second than you'd expect with
...
+
+1. Manually delete anything left over from RPM
+- `sudo rm -rf /usr/local/fusionauth/fusionauth-search/`
+- `sudo rm -rf /usr/local/fusionauth/fusionauth-app/`
+
+1. After removal, the following files will remain:
+* `/usr/local/fusionauth/config`
+* `/usr/local/fusionauth/data`
+* `/usr/local/fusionauth/java`
+* `/usr/local/fusionauth/bin`
```
[the remark this answers is not in the document]
Comment (mooreds) [reply site/docs/v1/tech/troubleshooting.adoc]: You can actually add a troubleshooting section and extract out this block of text to a include file, the way `_email-troubleshooting.adoc` is extracted and included in two places.

I think that `bin` should be removed as that will be installed again with the RPM, I believe.

[the remark this answers is not in the document]
Comment (jobannon) [reply site/docs/v1/tech/troubleshooting.adoc]: Good idea!💡 Broke out and updated to reflect a `bin` removal

**Comment by jobannon** [comment]
@robotdan, merging today 🙂

**Pull request merged**